### PR TITLE
improved query function behavior

### PR DESCRIPTION
**Pull request by ErikBjare**
Returns the only timeperiod if only one was given (i.e. `start` and `end` was used).

### CODE REVIEW

File: aw_client/client.py [hunk 1]
```diff
@@ -235,23 +235,37 @@ def setup_bucket(self, bucket_id: str, event_type: str):
     #   Query (server-side transformation)
     #
 
-    def query(self, query: str, start: datetime, end: datetime, name: str=None, cache: bool=False) -> Union[int, dict]:
+    def query(self,
+              query: str,
+              start: datetime = None,
```
[the remark this answers is not in the document]
Comment (ErikBjare) [reply aw_client/client.py]: I think it's so we're not breaking backward compatibility.

[the remark this answers is not in the document]
Comment (johan-bjareholt) [reply aw_client/client.py]: No-one is really using aw-client yet though?

File: aw_client/client.py [hunk 2]
```diff
@@ -235,23 +235,37 @@ def setup_bucket(self, bucket_id: str, event_type: str):
...
-            'timeperiods': ["/".join([start.isoformat(), end.isoformat()])],
+            'timeperiods': ["/".join([t[0].isoformat(), t[1].isoformat()]) for t in timeperiods],
             'query': query.split("\n")
         }
         response = self._post(endpoint, data, params=params)
         if response.text.isdigit():
             return int(response.text)
         else:
-            return response.json()
+            if len(timeperiods) > 1:
```
Comment (johan-bjareholt) [inline aw_client/client.py]: I'm not a fan of this assumption, I think it's better to be consistent. It's very easy to code bugs with behaviors like this.

Comment (ErikBjare) [reply aw_client/client.py]: True, will fix.

File: aw_client/client.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ErikBjare) [reply aw_client/client.py]: I am. We could emit a deprecation warning in the future, then remove it.

[the remark this answers is not in the document]
Comment (johan-bjareholt) [reply aw_client/client.py]: Alright!

**Comment by ErikBjare** [comment]
Closing, redundant after fixes in #51

**Pull request closed without merging**